postixdroid/app/src/main/java/de/ccc/events/postixdroid/ui/MainActivity.java

```diff
@@ -684,4 +700,18 @@ public void onClick(DialogInterface dialogInterface, int i) {
         dialog.show();
         etInput.requestFocus();
     }
+
+    private void playSound(Integer resourceId) {
+        // DHL Mode will disable scan-beep and replace successful scan with DHL-beep
+        if (config.getDHLEnabled())
+            if (resourceId == R.raw.enter) {
+                resourceId = R.raw.dhl;
+            } else if (resourceId == R.raw.beep) {
+                return;
+        }
```

Comment:
what cursed indentation with missing {} is this! 😂

Reply:
Extra spicy; fixed